### ISSUE TITLE
About the weights of original CLIP layer

### ISSUE THREAD

**Issue by LLFabiann** [farewellthree/STAN#8]
Are the weights of original CLIP layer always frozen during the whole training process?

**Comment by farewellthree** [comment]
No, the original CLIP layers are open.  Ablation shows freezing CLIP leads to inferior results (but still owns obvious performance over baseline). To my best knowledge, there are **no** video-text retrieval models freezing CLIP, hence, we have to open CLIP for fair comparison.
